DogWalkerMVC/DogWalkerMVC.csproj

```diff
@@ -10,10 +10,6 @@
     <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="3.1.1" />
   </ItemGroup>
 
-  <ItemGroup>
-    <Folder Include="Views\Neighborhood\" />
-  </ItemGroup>
-
 
```

Comment:
I'm not sure if you want to remove this folder. This is being used, right?

Reply:
updated!